.travis.yml

```diff
@@ -32,16 +32,18 @@ jobs:
       script:
         - npm run test:ci
         - bash <(curl -s https://codecov.io/bash) -X gcov
-    - stage: e2e
-      name: Test Suite appNavigation&search
-      script: npm run build.e2e && SUITE="--suite authentication,listViews,navigation,application,pagination,search" npm run e2e:docker
-    - name: Test Suite actionsAvailable
-      script: npm run build.e2e && SUITE="--suite actionsAvailable" npm run e2e:docker
-    - name: Test Suite addRemoveContent
-      script: npm run build.e2e && SUITE="--suite addRemoveContent" npm run e2e:docker
-    - name: Test Suite manageContent
-      script: npm run build.e2e && SUITE="--suite manageContent" npm run e2e:docker
-    - name: Test Suite sharingContent&markFavorite
-      script: npm run build.e2e && SUITE="--suite sharingContent" npm run e2e:docker
-    - name: Test Suite viewContent&metadata&extensions
-      script: npm run build.e2e && SUITE="--suite viewer,infoDrawer,extensions" npm run e2e:docker
+    # - stage: e2e
+    #   name: Test Suite appNavigation&search
+    #   script: npm run build.e2e && SUITE="--suite authentication,listViews,navigation,application,pagination,search" npm run e2e:docker
+    - name: IDS Test Suite appNavigation&search
+      script: npm run build.e2e && ./start-sso.sh && SUITE="--suite authentication" npm run e2e
+    # - name: Test Suite actionsAvailable
```

Comment:
remove commented out code if it is no longer required

Reply:
Was only for faster testing. Reverted that